arrows/vtk/mesh_coloration.cxx

```diff
@@ -76,506 +88,596 @@ mesh_coloration::mesh_coloration()
+        kwiver::vital::rgb_color rgb = colorImage.at( x, y );
         bool showPoint = true;
-        if (has_mask)
+
+        if( has_mask )
         {
-          showPoint = (maskImage.at(x, y).r > 0);
+          showPoint = ( maskImage.at( x, y ).r > 0 );
         }
 
-        float depthBufferValue = 0;
-        if (remove_occluded_)
+        double depthBufferValue = 0;
+        if( remove_occluded_ )
         {
-          double* range = depthBuffer[idData].Range;
+          double* range = depthBuffer[ idData ].Range;
           float depthBufferValueNorm =
-            depthBuffer[idData].Buffer->GetValue(x + width * (height - y - 1));
-          depthBufferValue = range[0] + (range[1] - range[0]) * depthBufferValueNorm;
+            depthBuffer[ idData ].Buffer->GetValue(
+              static_cast< vtkIdType >( x + width * ( height - y - 1 ) ) );
+          depthBufferValue =
+            range[ 0 ] + ( range[ 1 ] - range[ 0 ] ) * depthBufferValueNorm;
         }
-        if ((! remove_occluded_ ||
-             depthBufferValue + occlusion_threshold_ > depth) &&
-            (! remove_masked_ || showPoint))
+        if( ( !remove_occluded_ ||
+              depthBufferValue + occlusion_threshold_ > depth ) &&
+            ( !remove_masked_ || showPoint ) )
         {
-          if (! all_frames_)
+          if( !all_frames_ )
           {
-            list0.push_back(rgb.r);
-            list1.push_back(rgb.g);
-            list2.push_back(rgb.b);
+            list0.push_back( rgb.r );
+            list1.push_back( rgb.g );
+            list2.push_back( rgb.b );
           }
           else
           {
-            unsigned char rgba[] = {rgb.r, rgb.g, rgb.b, 255};
-            perFrameColor[idData]->SetTypedTuple(id, rgba);
+            unsigned char rgba[] = { rgb.r, rgb.g, rgb.b, 255 };
+            perFrameColor[ idData ]->SetTypedTuple( id, rgba );
           }
         }
       }
-      catch(std::out_of_range const&)
+      catch ( std::out_of_range const& )
       {
         continue;
       }
     }
 
-    if (! all_frames_)
+    if( !all_frames_ )
     {
       // If we get elements
-      if (list0.size() != 0)
+      if( list0.size() != 0 )
       {
-        double const sum0 = std::accumulate(list0.begin(), list0.end(), 0);
-        double const sum1 = std::accumulate(list1.begin(), list1.end(), 0);
-        double const sum2 = std::accumulate(list2.begin(), list2.end(), 0);
-        double const nb_val = list0.size();
-        meanValues->SetTuple3(id, sum0 / (double)nb_val, sum1 / (double)nb_val, sum2 / (double)nb_val);
+        double const sum0 = std::accumulate( list0.begin(), list0.end(), 0 );
+        double const sum1 = std::accumulate( list1.begin(), list1.end(), 0 );
+        double const sum2 = std::accumulate( list2.begin(), list2.end(), 0 );
+        double const nb_val = static_cast< double >( list0.size() );
+        meanValues->SetTuple3( id, sum0 / (double) nb_val,
```

Comment:
I'm not going to let it hold up this PR, but I don't think the C-style casts to `double` are doing anything here because `nb_val` is declared as `double` right above this.

Reply:
Right; thanks. I changed `nb_val` to a `double` because it's always used that way, but missed that these casts are no longer needed. Fixed in #1294.